src/services/catalog.service.ts

```diff
@@ -152,6 +166,56 @@ export class CatalogService {
 
     return retCategories;
   }
+
+  /*
+   * Catalog service dynamically adds 'Other' main and sub categories.
+   * This method removes all 'Other' categories.  Used for unit testing
+   */
+  public removeOtherCategories() {
```

Comment:
Fixed, moved function into unit test.  I originally left it here because I think in the future, if we ever get to watching & dynamically updating items & categories, this function would be useful to reset the categories.